ui/src/components/review/emissions-widget/emissions-widget.tsx

```diff
@@ -243,8 +238,42 @@ const EmissionsWidget: FunctionComponent<Props> = (props) => {
     }
   }
 
+  // handle download notification
   
-
+  const handleDownload = (actorID: string, fileType: string) => {
```

Comment:
Would be good to validate here if fileType is one of the allowed types (either at runtime or with a string enum). You will get an API error otherwise but it's always nice to catch errors early.